src/include/distributed/commands.h

```diff
@@ -25,6 +25,8 @@ extern bool AddAllLocalTablesToMetadata;
 
 /* controlled via GUC, should be accessed via EnableLocalReferenceForeignKeys() */
 extern bool EnableLocalReferenceForeignKeys;
+extern bool EnableUnsafeTriggers;
```

Comment:
very minor:

Maybe leave an empty line in between them to prevent readers getting confused about whether the comment above is valid for `EnableUnsafeTriggers` too or not (indeed, it is not)

```suggestion

extern bool EnableUnsafeTriggers;
```